db/tests/resources/filter_sort_create.sql

```diff
@@ -0,0 +1,131 @@
+CREATE TABLE "FilterSort" (
+    mathesar_id integer NOT NULL,
+    "varchar" varchar(32),
+    "numeric" integer,
+    "date"    date,
+    "array"   integer[]
+);
+
+CREATE SEQUENCE "filter_sort_mathesar_id_seq"
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+ALTER SEQUENCE "filter_sort_mathesar_id_seq" OWNED BY "FilterSort".mathesar_id;
+
+ALTER TABLE ONLY "FilterSort" ALTER COLUMN mathesar_id SET DEFAULT nextval('"filter_sort_mathesar_id_seq"'::regclass);
+
+INSERT INTO "FilterSort" VALUES
```

Comment:
Couldn't you load from the CSV instead of duplicating the data? If you change the data, you'll need to change it in two places.

Reply:
Yep, will remove the CSV as the other file is also referenced as SQL.